localstack/services/awslambda/invocation/docker_runtime_executor.py

```diff
@@ -181,7 +248,7 @@ def start(self, env_vars: dict[str, str]) -> None:
 
     def stop(self) -> None:
         CONTAINER_CLIENT.stop_container(container_name=self.id, timeout=5)
-        CONTAINER_CLIENT.remove_container(container_name=self.id)
+        # CONTAINER_CLIENT.remove_container(container_name=self.id)
```

Comment:
nit: Is this change intended? (as it seems unrelated to the PR)

Reply:
good catch, this was indeed just a leftover :grin: